src/auth.ts

```diff
@@ -406,25 +419,53 @@ async function respondWithFile(
   req.socket.destroy();
 }
 
-async function loginWithoutLocalhost(userHint?: string): Promise<UserCredentials> {
-  const callbackUrl = getCallbackUrl();
-  const authUrl = getLoginUrl(callbackUrl, userHint);
+function urlsafeBase64(base64string: string) {
+  return base64string.replace(/\+/g, "-").replace(/=+$/, "").replace(/\//g, "_");
+}
+
+const authProxyClient = new apiv2.Client({
+  urlPrefix: api.authProxyOrigin,
+  auth: false,
+});
```

Comment:
For some reason, this is making the emulator tests fail? I'm not sure why this isn't working in those tests (but works fine in practice)